Include pypi-friendly metadata in pyproject.toml

malyvsen opened the pull request:
Will :rainbow: hopefully :rainbow: make the page on pypi look pretty on each new release.

malyvsen committed:
doc: include pypi-friendly metadata in pyproject.toml

samedii reviewed (commented):
Nice! There are some more classifiers that we use here (Jim added them)

https://github.com/Aiwizo/pytorch-datastream/blob/master/setup.cfg

malyvsen committed:
doc: add pypi classifiers

the ones related to programming language or license are automatically added by poetry

samedii reviewed (approved):
Inconsisent number of white spaces maybe but I'll approve
